x/oracle/client/cli/txRegisterOracle.go

```diff
@@ -101,6 +138,16 @@ func CmdRegisterOracle() *cobra.Command {
 		},
 	}
 
+	cmd.Flags().String(flagOracleUniqueID, "", "unique ID for oracle")
```

Comment:
And you can set required values as below.
```go
if err := cmd.MarkFlagRequired(flagOracleUniqueID); error != nil {
    panic (error)
}
```

Reply:
Thanks, I fixed it.